src/components/StakingHierarchy.js

```diff
@@ -0,0 +1,465 @@
+`
+
+const Content = styled.div`
+  grid-area: content;
+  margin: 1rem 0 3rem;
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    margin: 0;
+  }
+`
+
+const FlexCentered = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`
+
+const Glyph = styled(FlexCentered)`
+  grid-area: glyph;
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    grid-area: content;
+    svg {
+      width: 50%;
+      height: 50%;
+      opacity: 0.1;
+    }
+  }
+`
+
+const Ether = styled(FlexCentered)`
+  grid-area: ether;
+  z-index: 2;
+  max-width: 5rem;
+  margin: 0 auto;
+`
+
+const StyledEtherSvg = styled(EtherSvg)`
+  --size: ${({ size }) => size};
+  width: var(--size);
+  height: var(--size);
+`
+
+const Line = styled.aside`
+  grid-column: 1;
+  grid-row: 1/3;
+  width: 100%;
+  height: 100%;
+  text-align: center;
+  font-size: 28px;
+  position: relative;
+  &::after {
+    content: "";
+    height: calc(100% - 50px);
+    border-left: solid 4px orange;
+    position: absolute;
+    left: calc(50% - 2px);
+    top: 50px;
+    z-index: 1;
+  }
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    display: none;
+  }
+`
+
+const Gold = styled.span`
+  color: ${({ theme }) => theme.colors.stakingGold};
+  font-weight: 600;
+`
+
+const StakingHierarchy = () => {
+  return (
+    <Container>
+      <Section number="1">
+        <Ether>
+          <StyledEtherSvg size="100%" />
+        </Ether>
+        <Line />
+        <Header>
+          <h2>Solo home staking</h2>
+          <Pills className="subtext">
+            <p>
+              <em>Most impactful</em>
+            </p>
+            <p>Full control</p>
+            <p>Full rewards</p>
+            <p>Trustless</p>
+          </Pills>
+        </Header>
+        <Glyph>
+          <SoloGlyph />
+        </Glyph>
+        <Content>
+          <p>
+            Solo staking on the Ethereum Beacon Chain is the{" "}
+            <Gold>gold standard</Gold> for staking. It is trustless, provides
+            full participation rewards, and improves the decentralization of the
+            network.
+          </p>
+          <p>
+            Those considering solo staking should have at least 32 ETH, hardware
+            connected to the internet ~24/7, and some technical know-how.
+          </p>
+          <p>
+            Over time, tooling has improved to help users get past some of the
+            technical hurdles, including options to bypass the command-line
+            altogether, if that's not your thing.
+          </p>
+          <p>
+            Check out some solo staking tools, then get started staking on the
+            Prater Testnet over on the{" "}
+            <Link to="https://prater.launchpad.ethereum.org">
+              Staking Launchpad
+            </Link>
+            .
+          </p>
+          <ButtonLink
+            to="/staking/solo/"
+            onClick={() => {
+              trackCustomEvent({
```

Comment:
Are these custom event necessary? It's just clicking a link to navigate to a page.

Reply:
Same as above